docs/reference/streamfield/blocks.rst

```diff
@@ -6,11 +6,12 @@ StreamField block reference
 This document details the block types provided by Wagtail for use in :ref:`StreamField <streamfield>`, and how they can be combined into new block types.
 
 
-.. class:: wagtail.fields.StreamField(blocks, blank=False, min_num=None, max_num=None, block_counts=None)
+.. class:: wagtail.fields.StreamField(blocks, use_json_field=None, blank=False, min_num=None, max_num=None, block_counts=None, collapsed=False)
```

Comment:
I put `use_json_field=None` here because we also define it explicitly after the `blocks` argument in the `__init__` itself.